Dockerfile

```diff
@@ -4,7 +4,7 @@ ENV LANGUAGE en_US:en
 ENV LC_ALL en_US.UTF-8
 RUN zypper -n addrepo https://download.opensuse.org/repositories/devel:/languages:/erlang/SLE_15_SP3/devel:languages:erlang.repo
 RUN zypper -n --gpg-auto-import-keys ref -s
-RUN zypper -n in elixir
+RUN zypper -n in elixir cargo
```

Comment:
NITPICK: to be more explicit we are installing bot rust and cargo so: `rust cargo`

Reply:
ok :+1: